src/store/plugins/fungibleTokens.js

```diff
@@ -117,6 +121,21 @@ export default (store) => {
           option,
         );
       },
+      async getTokensHistory({ rootState, rootGetters: { activeNetwork, account }, commit }) {
+        const { address } = account;
+        const rawTransactions = ([
+          ...(await fetchJson(`${activeNetwork.middlewareUrl}/aex9/transfers/to/${address}`)),
+          ...(await fetchJson(`${activeNetwork.middlewareUrl}/aex9/transfers/from/${address}`)),
+        ]);
+
+        const transactions = await Promise.all(
```

Comment:
why do you still need to get the full tx? I see that this endpoint: https://testnet.aeternity.io/mdw/aex9/transfers/to/ak_2AVeRypSdS4ZosdKWW1C4avWU4eeC2Yq7oP7guBGy8jkxdYVUy is updated as https://github.com/aeternity/ae_mdw/issues/198 is fixed

Reply:
On the Details screen you would be able to see the details